sklearn_pandas/features_generator.py

```diff
@@ -34,9 +38,15 @@ def gen_features(columns, classes=None):
     for column in columns:
         feature_transformers = []
 
+        arguments = {}
+        if prefix and prefix != "":
```

Comment:
Given that `bool("")` is `True` in Python, the check for `and prefix != ""` is not necessary. Same for `suffix`.